src/promotion-builder/features/search-promotions/main-layout/Search.js

```diff
@@ -0,0 +1,27 @@
+
+import React from 'react';
+import PropsTypes from 'prop-types';
+
+import SearchForm from '../../search-promotions/sub-components/searchform/SearchForm';
+import SearchResult from '../../search-promotions/sub-components/searchresult/SearchResult';
+
+import './search.css';
+
+const Search = ({searchTerm, promotionData, onInputChange, editBtnClick, view }) => {
+    if(view==='Search'){
+        return(<div className="searchContainer">
+        <SearchForm value={searchTerm} onChange={onInputChange}/>
+        <SearchResult searchTerm={searchTerm} promotionData={promotionData}  editBtnClick={editBtnClick}/>
+    </div>)
+    }
+    return null;
```

Comment:
I don't think return null is needed anymore

Reply:
It was not working without return null